okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -270,6 +274,29 @@ SSLSocketFactory createSocketFactory() {
     }
   }
 
+  private static SSLContext getSslContext() throws NoSuchAlgorithmException {
+    if (GrpcUtil.IS_RESTRICTED_APPENGINE) {
+      try {
+        SSLContext tlsContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
+        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        keyStore.load(null, null);
```

Comment:
@nmittler So, I'm going to presume that means you don't need me to get this submitted, do you have an ETA for getting a release with this?  They released the required changes to Java 7 GAE Std - after telling me that weren't going to, so this appears to be the missing piece.  Once you release, I'll ask both the Veneer and Bigtable teams to update, then everything should just work.